utils/rpms/daos.spec

```diff
@@ -15,7 +15,7 @@
 
 Name:          daos
 Version:       2.6.2
-Release:       2%{?relval}%{?dist}
+Release:       3%{?relval}%{?dist}
```

Comment:
Well, oops. https://github.com/daos-stack/daos/commit/c631702067a1528b66445e84a4e9da998d4cd043 did not bump the `Release` (sorry, I merged that). So this should be `4` to be inline with the changelog comments. I guess we'll just skip `3`. @brianjmurrell Does that sound reasonable? Or should we keep the `Release: 3` and update the changelogs to be duplicated with `3` as well?

Reply:
Yeah, skip 3 here but leave it in the changelog(s).  As usual, keep the `debian/changelog` and the `daos.spec` in parity though.